build/base.sh

```diff
@@ -68,7 +68,11 @@ BASE_OBJDIR="$(make -C${SRCDIR}/release -V .OBJDIR)"
 setup_stage "${BASE_OBJDIR}/${BASE_DISTDIR}"
 
 # remove older object archives, too
-BASE_OBJ=$(make -C${SRCDIR}/release -V .OBJDIR)/base.txz
+if [ ${PRODUCT_HOST} != ${PRODUCT_ARCH} ]; then
+        BASE_OBJ=$(make -C/usr/obj${SRCDIR}/${PRODUCT_TARGET}.${PRODUCT_ARCH}/release -V .OBJDIR)/base.txz
+else
+        BASE_OBJ=$(make -C${SRCDIR}/release -V .OBJDIR)/base.txz
```

Comment:
I was thinking maybe this breaks because we do not pass ${MAKE_ARGS} here?

Reply:
yes, seems to work in a first test, but i also want to test this on a new build vm.

Maybe it's a good idea to pass the MAKE_ARGS everywhere?

Reply:
yep, anywhere 'make.*SRCDIR' is used:

    % git grep 'make.*SRCDIR' | grep -v MAKE_ARGS
    build/base.sh:BASE_DISTDIR="$(make -C${SRCDIR}/release -V DISTDIR)/${SELF}"
    build/base.sh:BASE_OBJDIR="$(make -C${SRCDIR}/release -V .OBJDIR)"
    build/base.sh:BASE_OBJ=$(make -C${SRCDIR}/release -V .OBJDIR)/base.txz
    build/kernel.sh:KERNEL_DISTDIR="$(make -C${SRCDIR}/release -V DISTDIR)/${SELF}"
    build/kernel.sh:KERNEL_OBJDIR="$(make -C${SRCDIR}/release -V .OBJDIR)"
    build/kernel.sh:KERNEL_OBJ=$(make -C${SRCDIR}/release -V .OBJDIR)/kernel.txz
    build/kernel.sh:DEBUG_OBJ=$(make -C${SRCDIR}/release -V .OBJDIR)/kernel-dbg.txz